index.ts

```diff
@@ -373,6 +357,12 @@ class Analytics {
       callback(err, data);
     };
 
+    const timeout = this.timeout
```

Comment:
figured i'd fix up the ts ignore statement while i was here, but it is unrelated to this pr

Reply:
revert or fix the bug w spreading ints

Reply:
we've typed this as `private timeout: number | null;` inside our constructor. The code inside flush handles the case where `this.timeout` is a string:
```
typeof this.timeout === 'string'
        ? ms(this.timeout)
        : this.timeout
```
I'll be removing this statement since it is redundant. We could move the check into the constructor itself, but I don't think it's necessary (maybe I'm misunderstanding how typing works though).